profiles/common/modules/features/cce_basic_config/cce_basic_config.info

```diff
@@ -60,6 +68,12 @@ dependencies[] = workbench
 dependencies[] = workbench_moderation
 dependencies[] = wysiwyg
 stylesheets[all][] = cce_basic_config.css
+# The image_captcha_fonts variable is set in cce_basic_config_strongarm_alter().
+# Please do not remove this entry when re-exporting the variables.
+features[variable][] = image_captcha_fonts
+# The print_pdf_pdf_tool variable is set in cce_basic_config_strongarm_alter().
+# Please do not remove this entry when re-exporting the variables.
```

Comment:
This comment should be present in a README file too because I do not think people read comment in such file.